### PR TITLE
Add Package.swift

### PR DESCRIPTION
**Pull request by timdonnelly**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,10 +27,23 @@ matrix:
...
       osx_image: xcode10.1
       before_install:
-      - gem update --system
-      - gem install bundler
+        - gem update --system
+        - gem install bundler
       script:
-      - bundle exec pod lib lint Workflow.podspec --allow-warnings
+        - bundle exec pod lib lint Workflow.podspec --allow-warnings
+    - language: swift
```
Comment (bencochran) [inline .travis.yml]: Could we name these jobs to make differentiating between the two Xcode builds easier?

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        .library(
+            name: "Workflow",
+            targets: ["Workflow"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/ReactiveCocoa/ReactiveSwift.git", from: "4.0.0")
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
```
Comment (bencochran) [inline Package.swift]: nit: remove template comments

File: .travis.yml (hunk 1 above)
Comment (timdonnelly) [reply .travis.yml]: Done!

File: Package.swift (hunk 2 above)
Comment (timdonnelly) [reply Package.swift]: Done!

**Pull request merged**